crates/stc_ts_file_analyzer/tests/tsc/types/nonPrimitive/.1.ts

```diff
@@ -0,0 +1,9 @@
+// @strict: true
```

Comment:
Seems like this test is ignored?

Reply:
Please unignore this, or delete `1.tsc-errors.json`

Reply:
sir, this ignored test file mean decrease matched error

Reply:
I will remove errors.json, thanks